CharacterCreation/Class-Specific Documentation/Engineer/Engineer Repair Processes.txt

```diff
@@ -0,0 +1,463 @@
+ally makes a lockpick, hacking, explosives, repair, heal, or tracking check, 
+they get a +15 to it. 
+
+== Regression Override  == 
+Potential Gain:60
+Action:     1 Action
+Duration:   8 Rounds
+Range:      Touch
+
+    You temporarily disable your cybernetic ally's regression process,
+the part of their personality program that slows down to triple check the 
+results of their actions. This sends them into a rage mode. They still know
+friend from foe, and they still have a janky understanding of their higher-
+order skills like lockpicking, hacking, or social finesse, but they have no
+fear, no pain, no remorse and no hesitation for about half of a minute. Your
+ally Cannot feel shock during this duration, and cannot be repaired (they
+won't stand still). 
+
+    Your Cybernetic Ally gets a bonus +3 to accuracy nearer than 41m. They also 
+deal an extra 20 damage with melee hits and may aim as a free action with a 
+ranged weapon as long as they pass a REF check of DC 60. They take a penalty of 
+-25% to skill checks, including acrobat or athletics checks (these take 
+regression to calculate). An ally affected by Regression Override may use a 
+Strength +2 bonus at any time, but each time they do, they take 20 Internal 
+Damage of wear and tear. An Engineer can only perform a Regression Override 
+once per Encounter. Regression Overrides do not stack.
+
+== System Restore   == 
+Potential Gain:35
+Action:     2 Actions
+Duration:   Permanent
+Range:      Touch
+
+    Can only be used on a cybernetic subject whose primary data center is still 
+intact that has "bled out" less than 8 Rounds ago. The Engineer makes a DC 60 
+Repair check. If successful, the subject is now "bleeding out". If the subject 
+is saved, they get the feat "Refurbished". Characters with the "Refurbished" 
+feat have a permanent -1 to Intelligence and can never again be revived with 
+System Restore.
+
+===== Material/Mechanic =====
+
+== Duct Tape    == 
+Potential Gain:40
+Action:     2 Actions
+Duration:   N/A
+Range:      Touch
+
+    You must have your Extruder drawn in order to use this ability. This process
+only works on a cybernetic patient. Your patient heals an ammount of health 
+listed on the table below. As your character learns more repair skill, the 
+better they heal. Unless otherwise stated, this ability cannot heal a patient 
+above half health.
+
++-----------------+--------------+--------+
+| Med Skill Above | Base Healing |  Dice  |
++-----------------+--------------+--------+
+|       9         |     20       | + 2d10 |
++-----------------+--------------+--------+
+|       19        |     30       | + 3d10 |
++-----------------+--------------+--------+
+|       29        |     35       | + 4d10 |
++-----------------+--------------+--------+
+|       39        |     40       | + 5d10 |
++-----------------+--------------+--------+
+|       49        |     50       | + 6d10 |
++-----------------+--------------+--------+
```

Comment:
So, and this applies to the Doctor tables too, I just think it would look cleaner if the first column just read Med Skill and had entries 10, 20, 30, 40, 50. I don't think that would be misinterpreted.

Oh, and I think you meant Repair Skill on the Engineer tables.